### DISCUSSION TITLE
Gopro 12 as camera trap

### DISCUSSION THREAD

**Discussion by Tahca** [gopro/labs#841] [Q&A]
Hi everyone, I'm trying to use a Gopro Hero 12 as a camera trap. I'm using "Video Motion Detection" of Gopro Labs, but it is working only one time. I need to use it for some hours more and more times.
Someone have a suggestion for me? many thanks, Roberto

**Comment by dnewman-gpsw** [comment]
Need to select Repeat.  Commands that end with !R will repeat.

**Reply by Tahca** [reply]
Yes, I tried, but didn't work..

**Reply by dnewman-gpsw** [reply]
You need share the full details of the command(s) you are trying, and the video modes you are trying it with.  There is no known bug with this feature.

**Reply by Tahca** [reply]
Thank you for your reply. Do you think that the configuration can help?
https://gopro.github.io/labs/control/set/?cmd=!SM5D12H7!R&title=Motion%20Detection

**Reply by dnewman-gpsw** [reply]
Not enough, as that is working perfectly here.  What is particular to your setup, what video mode do you want to use, what firmware version etc.?

**Reply by Tahca** [reply]
ok, I collected the info:
Serial number C3501325255737
Firmware Version 02.10.70
Video: 8:7 or 16:9 - 5,3K - 30fps

I hope it was enough, thanks..

**Reply by Tahca** [reply]
Hi, I done other tests but it's working only one time. After the camera need a new QR-code. Are the datas about the camera enough for understand the bug?

**Reply by dnewman-gpsw** [reply]
Update your firmware to the current version and try again please.

**Comment by Tahca** [comment]
ok, done... and then do I have to install again Gopro Labs on the camera? Now the camera give a message "QR Code not valid"...

**Reply by dnewman-gpsw** [reply]
What firmware version is showing in your camera's About menu?

**Reply by Tahca** [reply]
now 02.20

**Reply by dnewman-gpsw** [reply]
Yes, that is not a Labs version, so it will reject the QR codes that are designed for Labs.  Labs can only be installed manually, it cannot be installed by the Quik app.

**Comment by Tahca** [comment]
Right! now it seems working! I'll do some tests... Many thanks!
